src/platform/testing/e2e/cypress/support/index.d.ts

```diff
@@ -0,0 +1,110 @@
+/// <reference types="cypress" />
+
+/**
+ * Add custom command interfaces here to get intellisense and type checking
+ */
+declare namespace Cypress {
+  interface Chainable<Subject> {
+    /**
+     * Checks the passed selector and children for axe violations.
+     * @param {string} [context=main] - CSS/HTML selector for the container element to check with aXe.
+     * @param {Object} [tempOptions={}] - Rules object to enable _13647 exception or modify aXe config.
+     */
+    axeCheck(): Chainable<Subject>;
+
+    fill(selector: string, value: any): Chainable<Subject>;
+    fillName(baseName: string, name: object): Chainable<Subject>;
+    fillAddress(baseName: string, address: object): Chainable<Subject>;
+
+    /**
+     * Works with Date widget. And va-date and va-memorable-date web components;
+     * expects dateString in YYYY-MM-DD format
+     */
+    fillDate(fieldName: string, dateString: string): Chainable<Subject>;
```

Comment:
the code in here (preexisting method), has some overlap with the new fillMemorableDateWebComponent. I can look into refactoring further if you think I should.